api/database/seeders/UserSeederUat.php

```diff
@@ -16,32 +16,187 @@ class UserSeederUat extends Seeder
+            ['sub' => 'c0967038-f2bb-42cc-8bb3-4b90e9f77f89'],
             [
-                'first_name' => 'tristan-orourke',
-                'last_name' => 'Talent',
-                'sub' => 'b4c734a1-dcf3-4fb8-a860-c642700cb0b8',
+                'first_name' => 'Wanda',
+                'last_name' => 'Maximoff',
+                'email' => 'usertester923@gmail.com',
                 'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
             ]
         );
         User::updateOrCreate(
-            ['email' => 'petertgiles'.$fakeEmailDomain],
+            ['sub' => 'b4c734a1-dcf3-4fb8-a860-c642700cb0b8'],
             [
-                'first_name' => 'petertgiles',
-                'last_name' => 'Talent',
-                'sub' => '5d0ce9a8-8164-46a5-9fe1-b7a2b42d61fc',
+                'first_name' => 'Tristan',
+                'last_name' => "O'Rourke",
+                'email' => "tristan.o'rourke@tbs-sct.gc.ca",
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['sub' => '5d0ce9a8-8164-46a5-9fe1-b7a2b42d61fc'],
+            [
+                'first_name' => 'Peter',
+                'last_name' => 'Giles',
+                'email' => 'peter.giles@tbs-sct.gc.ca',
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['sub' => 'fcd5d233-66d9-4ac6-81e0-fabf379b0a70'],
+            [
+                'first_name' => 'Biruntha',
+                'last_name' => 'Sasikumar',
+                'email' => 'biruntha.sasikumar@tbs-sct.gc.ca',
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['sub' => 'b9305997-b221-43f7-9c25-bc0d460b6464'],
+            [
+                'first_name' => 'Maddy',
+                'last_name' => 'Test',
+                'email' => 'testing@gmail.com',
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['sub' => 'a5bf84fe-7003-44df-bbdd-ce2d35cac2fb'],
+            [
+                'first_name' => 'Yonathan',
+                'last_name' => 'Kidanemariam',
+                'email' => 'Yonathan.Kidanemariam@tbs-sct.gc.ca',
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['sub' => '43a2b0f0-218b-447b-9208-9254b487ebab'],
+            [
+                'first_name' => 'Julie',
+                'last_name' => 'Harrison',
+                'email' => 'julie.harrison@tbs-sct.gc.ca',
+                'roles' => [ApiEnums::ROLE_ADMIN, ApiEnums::ROLE_APPLICANT]
+            ]
+        );
+        User::updateOrCreate(
+            ['sub' => '43a2b0f0-218b-447b-9208-9254b487ebab'],
```

Comment:
Could you double-check this one, please?  It has the same sub as the one before it.

Reply:
Nice Catch 💯 Fixed by [f42f784](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5313/commits/f42f7841bdd28485d24336f8abd8b0b62b39a781)

Reply:
:pray: